contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -1110,15 +1104,46 @@ contract MinterSEAV0 is ReentrancyGuard, MinterBase, IFilteredMinterSEAV0 {
     }
 
     /**
-     * @notice Transfer ETH. If the ETH transfer fails, wrap the ETH and send it as WETH.
+     * @notice Force sends `amount` (in wei) ETH to `to`, with a gas stipend
+     * equal to `minterRefundGasLimit`.
+     * If sending via the normal procedure fails, force sends the ETH by
+     * creating a temporary contract which uses `SELFDESTRUCT` to force send
+     * the ETH.
+     * Reverts if the current contract has insufficient balance.
+     * @param _to The address to send ETH to.
+     * @param _amount The amount of ETH to send.
+     * @dev This function is adapted from the `forceSafeTransferETH` function
+     * in the `https://github.com/Vectorized/solady` repository.
      */
-    function _safeTransferETHWithFallback(address to, uint256 amount) internal {
-        (bool success, ) = to.call{value: amount, gas: minterRefundGasLimit}(
-            ""
-        );
-        if (!success) {
-            weth.deposit{value: amount}();
-            weth.transfer(to, amount);
+    function _forceSafeTransferETH(address _to, uint256 _amount) internal {
+        // load state variable into memory for use in inline assembly
+        uint256 minterRefundGasLimit_ = minterRefundGasLimit;
+        // Manually inlined because the compiler doesn't inline functions with
+        // branches.
+        /// @solidity memory-safe-assembly
+        assembly {
+            // If insufficient balance, revert.
+            if lt(selfbalance(), _amount) {
+                // Store the function selector of `ETHTransferFailed()`.
+                mstore(0x00, 0xb12d13eb)
```

Comment:
Where is this selector defined?

Reply:
the function selector is calculated by hashing the string, but I don't know why storing it at memory location 0x00 would do anything since the next step is to immediately revert.
This entire check has been removed, so I'm okay not diving deeper here, but this specific operation doesn't make sense to me. I'll include a question about this as part of our code review response 🙏

Reply:
That SGTM ser – fantastic! TYVM!!

Reply:
ohhh, actually this makes sense -
- `mstore` stores 32 byes, so memory looks like: `0x00000000000000000000000000000000000000000000000000000000b12d13eb`
- revert starts at bytes 28 in memory, and has width of 4 bytes, so return data is simply the function selector of `ETHTransferFailed()`, `0xb12d13eb`

I'm guessing services like etherscan are smart enough to have common functions reverse-mapped, and this would probably show up on their site with a revert reason of `ETHTransferFailed()`